include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -1814,7 +1814,8 @@ namespace internal
                          dofs_per_line =
                            accessor.get_fe(fe_index).dofs_per_line;
       std::vector<types::global_dof_index>::iterator next = dof_indices.begin();
-      for (unsigned int vertex = 0; vertex < 2; ++vertex)
+      for (unsigned int vertex = 0; vertex < GeometryInfo<1>::vertices_per_cell;
+           ++vertex)
```

Comment:
Can you use the recently added range-based for loops here and elsewhere?
```suggestion
      for (const unsigned int vertex : GeometryInfo<1>::vertex_indices())
```

Reply:
Done.